src/app/utils/field-validation/validators/textValidator.ts

```diff
@@ -0,0 +1,80 @@
+import { chain, left, right } from 'fp-ts/lib/Either'
+import { pipe } from 'fp-ts/lib/pipeable'
```

Comment:
the `pipe` documentation says "Use pipe from function module instead.", not sure if that means we should be importing from `fp-ts/lib/function`